### PR TITLE
Fixed 2 buggs on create challenge

### PR DESCRIPTION
**Pull request by BQ-QB**
fixed toolbar color bugg
and text input borders and gravity, they could still be approved though.

### CODE REVIEW

**Comment by BQ-QB** [comment]
cred till wendy för hjälp!

**Review by Wendaaj — APPROVED** [review]

File: app/src/main/java/com/example/sonymz1/SplashFragment.java [hunk 1]
```diff
@@ -59,9 +59,9 @@ private void CheckLogin() {
         Integer userID = sp.getInt("id", -1);
 
         // Uncomment if you want to go to login page again.
-        //spEditor = sp.edit();
```
Comment (Fajlix) [inline app/src/main/java/com/example/sonymz1/SplashFragment.java:62]: This should be commented when pushed

**Review by Fajlix — CHANGES_REQUESTED** [review]
Just comment out the sp editor and everything will be good!

File: app/src/main/java/com/example/sonymz1/SplashFragment.java (hunk 1 above)
Comment (BQ-QB) [reply app/src/main/java/com/example/sonymz1/SplashFragment.java]: Fixat!

**Review by Fajlix — APPROVED** [review]
Good!

**Pull request merged**